### PR TITLE
Updated JenkinsFile to rm -Rf instead of -R as it was failing if the folder did not exist.

### PR DESCRIPTION
**Pull request by sharath-os**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -38,7 +38,7 @@ node('ios-osx') {
         sh '''#!/bin/bash -l
             set -x
             set -e
-            rm -R ./node_modules
+            rm -Rf ./node_modules
```
Comment (chrmod) [inline Jenkinsfile]: while using `yarn` deleting node_modules should not be necessary. Can we please try to remove it and see how it goes?

**Review by chrmod — DISMISSED** [review]

**Comment by sammacbeth** [comment]
The Prepare step is failing, can you have a look @sharath-cliqz ?

**Comment by sharath-os** [comment]
@sammacbeth Yes. It was failing due to the following error:
+ pod install
Analyzing dependencies
Fetching podspec for `RNDeviceInfo` from `./node_modules/react-native-device-info`
[!] No podspec found for `RNDeviceInfo` in `./node_modules/react-native-device-info`

Is it because i removed the 'rm -Rf ./node_modules' line as requested by @chrmod ??

**Comment by sharath-os** [comment]
Yes it seems to work fine if i leave the "rm -rf ./node_modules" in there.
@chrmod i guess it doesnt make sense to remove it as if i remove it, the script fails at prepare stage.
@sammacbeth @mahmoud-cliqz : Can you please review and merge ?
Thanks.

**Review by sammacbeth — APPROVED** [review]

**Pull request merged**